package Message; 
message BossInspireRequest					//ÇëÇó¹ÄÎè
{ 
   optional int64     pid 		= 1;  //½ÇÉ«ID
} 